README.md

```diff
@@ -1,59 +1,184 @@
 ```
 
-## Project Setup Notes
+Note: The `application.properties` file has the connection information to connect to your Capella cluster. You can also pass the connection information as environment variables to the Docker container.
+If you choose not to pass the environment variables, you can update the `application.properties` file in the `src/main/resources` folder.
+
+### Verifying the Application
+
+Once the application starts, you can see the details of the application on the logs.
+
+![Application Startup](./assets/images/app-startup-spring-boot.png)
+
+The application will run on port 8080 of your local machine (http://localhost:8080). You will find the interactive Swagger documentation of the API if you go to the URL in your browser. Swagger documentation is used in this demo to showcase the different API endpoints and how they can be invoked. More details on the Swagger documentation can be found in the [appendix](#swagger-documentation).
+
+![Swagger Documentation](./assets/images/swagger-documentation-spring-boot.png)
+## Running Tests
+
+To test your application, you can use Maven (mvn) to run the test suite. Here are the steps to run the tests:
+
+### Run All Tests:
+
+```sh
+mvn test
+```
+
+This command will execute all the test cases in your project.
+
+### Run Individual Tests:
+
+Additionally, you can run individual test classes or methods using the following commands:
+
+To run the tests for the AirlineIntegrationTest class:
+
+```sh
+mvn test -Dtest=org.couchbase.quickstart.springboot.controllers.AirlineIntegrationTest
+```
+
+To run the tests for the AirportIntegrationTest class:
+
+```sh
+mvn test -Dtest=org.couchbase.quickstart.springboot.controllers.AirportIntegrationTest
+```
+
+To run the tests for the RouteIntegrationTest class:
+
+```sh
+mvn test -Dtest=org.couchbase.quickstart.springboot.controllers.RouteIntegrationTest
+```
+
+## Appendix
+
+### Data Model
+
+For this quickstart, we use three collections, `airport`, `airline` and `routes` that contain sample airports, airlines and airline routes respectively. The routes collection connects the airports and airlines as seen in the figure below. We use these connections in the quickstart to generate airports that are directly connected and airlines connecting to a destination airport. Note that these are just examples to highlight how you can use SQL++ queries to join the collections.
+
+![travel-sample data model](/assets/images/travel_sample_data_model.png)
+
+### Extending API by Adding New Entity
+
+If you would like to add another entity to the APIs, these are the steps to follow:
+
+- Create the new entity (collection) in the Couchbase scope. You can create the collection using the [SDK](https://docs.couchbase.com/java-sdk/current/howtos/provisioning-cluster-resources.html#collection-management) or via the [Couchbase Server interface](https://docs.couchbase.com/cloud/n1ql/n1ql-language-reference/createcollection.html).
```

Comment:
Create the new entity (collection) in the Couchbase scope. You can create the collection using the -> Create a new collection in the Couchbase scope. You can create new collections using the ...

Reply:
changed it